### ISSUE TITLE
Multiple GPU

### ISSUE THREAD

**Issue by sarangs-ntnu** [centre-for-humanities-computing/stormtrooper#1]
I want to use the Generative part using multiplegpus to load the Beluga2 which is trained on LLAMA 2 70B,

**Comment by x-tabdeveloping** [comment]
Sorry for the late response. I'm currently working on release 1.0.0, which will contain the option of running on multiple GPUs.
I will keep you up to date on this thread. You can expect the new release within a week.

**Comment by x-tabdeveloping** [comment]
I released version 1.0.0, you can now run inference on multiple GPUs by loading models with `device_map="auto"`:
```python
from stormtrooper import Trooper

model = Trooper("petals-team/StableBeluga2", device_map="auto")
```
